src/ImageMetadata.jl

```diff
@@ -184,8 +175,11 @@ end
 Base.show(io::IO, img::ImageMeta) = showim(io, img)
 Base.show(io::IO, ::MIME"text/plain", img::ImageMeta) = showim(io, img)
 
-Base.reinterpret(::Type{T}, img::ImageMetaArray) where {T} = shareproperties(img, reinterpret(T, img.data))
-Base.reinterpret(::Type{T}, img::ImageMeta) where {T} = error("reinterpret method not defined for $(typeof(img)). Consider a MappedArray instead.")
+function ImageCore.reinterpret(::Type{T}, img::ImageMeta{T2,N,A}) where {T,T2,N,A}
+    (A <: Array) || error("reinterpret method not defined for $(typeof(img)). Consider a MappedArray instead.")
```

Comment:
I think the new `reinterpret` doesn't have this limitation anymore (?)
That said, I don't know this. This will need someone like Prof Holy to review (which I saw you already pinged, so I won't ping again)

Reply:
Correct, it's not a limitation anymore.